### PR TITLE
feat-6837: add helmfile documentation

### PR DESCRIPTION
**Pull request by thameezb**
Adds in documentation for helmfile addition to Spinnaker

Relies on https://github.com/spinnaker/deck/pull/9998

Related to https://github.com/spinnaker/spinnaker/issues/6837

### CODE REVIEW

**Comment by dbyron-sf** [comment]
For completeness, can you rebase to the HEAD of master please?

**Comment by dbyron-sf** [comment]
And also add something to the [next release preview](https://github.com/spinnaker/spinnaker.io/blob/1c4a755e6e69689d6dd493c042c1b777602686c7/content/en/docs/releases/next-release-preview/index.md) please?

File: content/en/docs/releases/next-release-preview/index.md
Comment (thameezb) [inline content/en/docs/releases/next-release-preview/index.md:78]: This link will only be live once we merge to master. I am not sure the approach taken by the code owners to self-referential references

File: content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md
Comment (dbyron-sf) [reply content/en/docs/guides/user/kubernetes-v2/deploy-helm/index.md]: Yeah, there probably isn't a 100% bulletproof way,  https://github.com/spinnaker/rosco/blob/master/Dockerfile.slim#L7 will likely stop pointing to the helmfile version at some point, but so be it.

**Review by dbyron-sf — APPROVED** [review]

**Pull request merged**
